### DISCUSSION TITLE
Monitoring on one page

### DISCUSSION THREAD

**Discussion by Feriman22** [louislam/dockge#354] [Feature Request]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

UI Feature

### 🔖 Feature description

Able to view the monitoring data from "docker stats"

### ✔️ Solution

View docker stats data in a graph, like Grafana, all container stats on one page.

### ❓ Alternatives

There are other solutions, like cAdvisor, influxdb, etc, but it would be great to have this feature in Dockge

**Comment by ryangurn** [comment]
This feature would be quite useful specifically since the goal of this project was to replace tools like portainer which provide extensive information that the docker engine (or kubernetes) is providing. 

Since we need to be realistic about things it would be nice to see some stats related to the container as well as some related to the docker engine (such as storage used, image counts, cpu, mem). Some of those things might be stuff to strive towards in the future but would all be nice nonetheless.

**Comment by trythatagain** [comment]
This would be a nice addition, at the moment I use  [Dozzle](https://dozzle.dev/)  to show the logs and some basic container and docker information.

**Reply by tippfehlr** [reply]
Dozzle is great, thanks.
The log viewer is already in Dockge (although it is fancier in Dozzle), but a small CPU/Mem graph would be great.
